components/aws/sagemaker/batch_transform/src/batch_transform.py

```diff
@@ -65,8 +64,8 @@ def main(argv=None):
   logging.info('Batch Job request submitted. Waiting for completion...')
   _utils.wait_for_transform_job(client, batch_job_name)
 
-  Path(args.output_location_file).parent.mkdir(parents=True, exist_ok=True)
-  Path(args.output_location_file).write_text(unicode(args.output_location))
+  with open('args.output_location_file', 'w') as f:
+    f.write(unicode(args.output_location))
```

Comment:
This seems like a bug fix that is unrelated to this PR?

Reply:
ya ... reverted